### PR TITLE
Use custom KMS key for encryption on your Amazon EBS volumes

### PR DESCRIPTION
**Pull request by njuettner**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by paurosello** [comment]
I think we need to allow access to all kms keys with "*" as they could swap the KMS keys anytime on AWS and then the cluster will use multiple, each EBS can be encrypted with a different key.

Also we can remove the conditional as it will always be true, we generate a KMS key by default

**Comment by njuettner** [comment]
That sounds good Pau, let's do that 👍🏻

**Review by njuettner — COMMENTED** [review]
looks good to me, unfortunately I can't approve my own PR :D

File: pkg/project/project.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "aws-operator"
 	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "14.0.1-dev"
+	version            = "14.0.1-iamrole"
```
Comment (whites11) [inline pkg/project/project.go:8]: this doesn't have to be merged, please

**Review by whites11 — APPROVED** [review]
LGTM

File: pkg/project/project.go (hunk 1 above)
Comment (paurosello) [reply pkg/project/project.go:8]: yeah, reverting back now

**Comment by paurosello** [comment]
tested on gauss and works

**Pull request merged**